hazelcast-jet-core/src/test/java/com/hazelcast/jet/core/JetTestSupport.java

```diff
@@ -125,6 +134,19 @@ protected static File createTempDirectory() throws IOException {
         return file;
     }
 
+    public static void assertJobStatusEventually(Job job, JobStatus expected) {
+        assertJobStatusEventually(job, expected, 10);
+
+    }
+
+    public static void assertJobStatusEventually(Job job, JobStatus expected, int timeoutSeconds) {
+        assertTrueEventually(() -> {
+            assertNotNull(job);
```

Comment:
what's the point of the null check here?

Reply:
Yes, you are right I'll pull that out of the block.

Reply:
https://github.com/hazelcast/hazelcast-jet/pull/1136